src/core/shape_inference/include/squeeze_shape_inference.hpp

```diff
@@ -90,6 +90,18 @@ std::vector<TRShape> shape_infer(const Squeeze* op,
                              });
             }
         } else {
+            if (op->get_pytorch_dynamic_rank()) {
+                int i{};
+                if (std::any_of(arg_shape.cbegin(), arg_shape.cend(), [&](const DimType& d) {
```

Comment:
In lambda capture what is required only.

Reply:
done